packages/disploy/src/structs/ContextMenuCommand.ts

```diff
@@ -8,7 +8,7 @@ export class ContextMenuInteraction extends CommandInteraction {
 	 */
 	public targetId: Snowflake;
 
-	public constructor(app: App, raw: APIContextMenuInteraction) {
+	public constructor(app: App, public override raw: APIContextMenuInteraction) {
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.